Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -59,6 +52,14 @@ def test_alerts_in_sentinel():
 if error:
     print("error --> %s" % error.decode())
 
+f = open('../cohesity.json',)
+data = json.load(f)
+resource_group = data['resource_group']
+workspace_name = data['workspace_name']
+api_key = data['apiKey']
```

Comment:
Why is apiKey in camel case but not resource_group or workspace_name? Please try to be consistent.

Reply:
b/c in helios endpoint,